test/globalSetup.ts

```diff
@@ -0,0 +1,4 @@
+module.exports = (): void => {
+  process.env.NETWORK = "rinkeby";
+  process.env.INFURA_API_KEY = "213kldjalpi";
```

Comment:
I guess we don't mind committing this?

Reply:
Of course not a real API key.
Those env vars are required to be set, but I don't want to have to set them when running the tests.
So, anything will do.